etc/column/column.conf

```diff
@@ -0,0 +1,10 @@
+[DEFAULT]
+# Log file
+log_file = /var/log/column.log
+
+# Log info level
+log_level = DEBUG
+
+[API]
+server = 0.0.0.0
```

Comment:
It's a security best practice.  Using 127.0.0.1 is much better.  And rarely if ever do deployers do the due diligence or even aware of the consequences.